content/js.md

```diff
@@ -6,40 +6,58 @@ be compiled, and `target` is the file that should be created (the path is, of
 course, modified a little when you use fingerprinting).
 
 
-## Transpilation
+## Format and Modules
 
-You can also transpile modern JS to JavaScript that works in older browsers. To
-use that, you need to install `faucet-pipeline-esnext` instead of
-`faucet-pipeline-js`. In addition, you set `esnext` to `true` for each bundle
-that you want to transpile. You also need a [`browserslist`
-file](https://github.com/ai/browserslist/) to specify which browsers you want to
-support. This is an example configuration:
+By default, faucet builds your file as an IIFE. You can also configure your
+bundle to output a different module format like ESM (your choices are: `iife`,
+`esm`, `umd`, `amd`, and `commonjs`):
 
 ```js
 module.exports = {
     js: [{
         source: "./index.js",
         target: "./dist/bundle.js",
-        esnext: true
+        format: "esm"
     }]
 };
 ```
 
-You can also exclude modules of the bundle you don't want to transpile with
-`exclude`. This is useful for dependencies that are already transpiled:
+If you chose a module format that does not support importing/exporting natively
+(i.e. `iife` or `umd`), the `exports` option determines the name of the global
+variable under which the source module's export will be exposed.
+
+If you want to include an external library that is delivered separately and
+exported to the global namespace, you can configure it as an "external" in
```

Comment:
This paragraph seems confusing if you don't already understand the mechanisms.

```suggestion
If your source code references an external library which you don't want
included in the bundle - typically because its API is provided as a global
variable - you can declare that library to be "external":
```

(The phrasing here is far from perfect though; consider it a rough draft.)

Reply:
I think your suggestion is much better, what do you not like about it?